src/main/java/Formatter.java

```diff
@@ -0,0 +1,113 @@
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.List;
+
+public class Formatter {
+
+    public static String parseUserInput(String[] inputLine, int startIndex, int endIndex) {
+        String[] updatedArray = Arrays.copyOfRange(inputLine, startIndex, endIndex);
+        return String.join(" ", updatedArray);
+
+    }
+
+    public static int indexOf(String[] userInput, String toFind) {
+        // if array is Null
+        if (userInput == null) {
+            return -1;
+        }
+        for (int i = 0; i < userInput.length; i++) {
+            if (userInput[i].equals(toFind)) {
+                return i;
+            }
+        }
+        return -1;
+    }
```

Comment:
Can consider using the default Java String indexOf() Method instead of implementing your own.